[IMP] accounting: internal transfers (#7208)
odoo/documentation: content/applications/finance/accounting/payments/internal_transfers.rst

Internal transfers
==================

Internal money transfers can be handled in Odoo. At least two bank accounts are needed to make
internal transfers.

.. seealso::
   :doc:`How to add an additional bank account <../bank>`

Configuration
=============

An internal transfer account is automatically created on your database based on your company's
localization and depending on your country’s legislation. If needed, the default :guilabel:`Internal
transfer account` can be modified by going to :menuselection:`Accounting --> Configuration -->
Settings --> Default Accounts section`.

Register an internal transfer from one bank to another
======================================================

Log an internal transfer
------------------------

> In order to transfer 1000 USD from Bank A to Bank B, from the Accounting Dashboard, click on the
> drop-down selection button (:guilabel:`⋮`) on one of your banks, then click on :guilabel:`Payments`
> and each payment can be marked as :guilabel:`Internal Transfer`. A :guilabel:`Destination Journal`
> must be selected in order to :guilabel:`Confirm` the internal transfer.

Reviewer: ```suggestion
In order to transfer 1000 USD from Bank A to Bank B, from the Accounting Dashboard, click on the
drop-down selection button (:guilabel:`⋮`) on one of your banks, then click on :guilabel:`Payments`
and each payment can be marked as :guilabel:`Internal Transfer`. A :guilabel:`Destination Journal`
must be selected in order to :guilabel:`Confirm` the internal transfer.
```
If you want to transfer 1000 USD from bank A to bank B, for example, from the Accounting dashboard, click the **drop-down** selection button (:guilabel:`⋮`) on one of your bank journals, and click :guilabel:`Payments`. Each payment can be marked as :guilabel:`Internal Transfer`, however, a :guilabel:`Destination Journal` must be selected in order to :guilabel:`Confirm` the internal transfer.

.. note::
   You can only make **internal transfers** on *unposted* payments.

Internal transfers
==================

Internal money transfers can be handled in Odoo. At least two bank accounts are needed to make
internal transfers.

.. seealso::
   :doc:`How to add an additional bank account <../bank>`

Configuration
=============

An internal transfer account is automatically created on your database based on your company's

> localization and depending on your country’s legislation. If needed, the default :guilabel:`Internal

Reviewer 2: ```suggestion
To modify the default :guilabel:`Internal
```

> transfer account` can be modified by going to :menuselection:`Accounting --> Configuration -->
> Settings --> Default Accounts section`.

Reviewer 2: ```suggestion
transfer account`, go to :menuselection:`Accounting --> Configuration --> Settings`, then scroll down to the :guilabel:`Default Accounts` section.
```

Register an internal transfer from one bank to another
======================================================

> Log an internal transfer

Reviewer 2: See my comment on the reconciliation section, but if you remove the reconciliation section of this doc, I would remove this sub-title as well as it would not be necessary.

------------------------

> If you want to transfer 1000 USD from Bank A to Bank B, from the Accounting Dashboard, click on the
> drop-down selection button (:guilabel:`⋮`) on one of your banks, then click on :guilabel:`Payments`.
> Each payment can be marked as :guilabel:`Internal Transfer`, however, a :guilabel:`Destination Journal`
> must be selected in order to :guilabel:`Confirm` the internal transfer.

Reviewer 2: I would rephrase this to a) remove the example part of the explanation as i don't feel it's necessary b) phrase the second part as more of a procedure:

```suggestion
If you want to transfer money from one bank to another, access the Accounting Dashboard, click the drop-down selection button (:guilabel:`⋮`) on the bank from which you want to make the transfer, then click guilabel:`Payments`. Select or create a payment, tick the :guilabel:`Internal Transfer` checkbox, and select a :guilabel:`Destination Journal` before you :guilabel:`Confirm` the internal transfer.
```

.. note::
  You can only make **internal transfers** on *unposted* payments.

.. image:: internal_transfers/transfer-check.png

>    :align: center

Reviewer 2: remove this

   :alt: Internal transfer activation

The money is now booked in the transfer account and another payment is automatically created in the
destination journal.

> Bank journal (Bank A)

Reviewer 2: I would put the tables in an example box.

Also, regarding the second title:
- Is the "Automated booking" part really necessary?
- it's a bit of a detail but I would remove the capital letters in "Bank B"

     - **$1,000**

.. note::

>    There is one outstanding payment and one outstanding receipt pending in your two bank account

Reviewer 2: If you do decide to have the tables in an example box, I'd remove the note and include the text in the example box after the tables.

   journals, because the bank statement confirming the sending and receiving of the money has not
   been booked yet.

> Internal transfer reconciliation

Reviewer 2: Is this whole section necessary? Is there anything in the reconciliation process that's specific to internal transfers? 
If not, I would maybe just add a sentence at the end of the previous section explaining that reconciliation is done as usual + keep the seealso linking to the reconciliation doc page.
